tests/models/test_onnx.py

```diff
@@ -152,3 +141,43 @@ def to_numpy(tensor):
 
     # compare ONNX Runtime and PyTorch results
     assert np.allclose(to_numpy(torch_out), ort_outs[0], rtol=1e-03, atol=1e-05)
+
+
+def test_model_saves_with_tuple_input(tmpdir):
+    """Test that ONNX model saves when input is tuple of tensors"""
+    class CustomModel(EvalModelTemplate):
```

Comment:
Better to use BoringModel as faster execution.

Reply:
done